src/org/omegat/Main.java

```diff
@@ -277,14 +281,26 @@ protected static int runGUI() {
             // do nothing
         }
 
+        String theme = Preferences.getPreferenceDefault(Preferences.THEME_SELECTED_NAME, "Default");
         try {
-            UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-        } catch (Exception e) {
-            // do nothing
+            if (theme.equals("Default")) {
```

Comment:
This deep nesting kind of makes me nervous.

Maybe we could move the LAF setting to a separate method.